<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SQLite2HTML</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblDataBase">
          <property name="text">
           <string>Datenbank:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leDataBase"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectDatabase">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Auswählen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblHtmlSavePath">
          <property name="text">
           <string>HTML Speicherpfad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leHtmlSavePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectHtmlSavePath">
          <property name="text">
           <string>Auswählen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblLinklistSavePath">
          <property name="text">
           <string>Linkliste Speicherpfad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLinklistSavePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectLinklistSavePath">
          <property name="text">
           <string>Auswählen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblProjectTitle">
          <property name="text">
           <string>Projekttitel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leProjectTitle"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lblState">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="pteState">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnStartConvert">
          <property name="text">
           <string>Umwandeln</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblVersion">
          <property name="text">
           <string>Version: 1.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblAuthor">
          <property name="text">
           <string>Autor: Voldracarno Draconor</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
